### DISCUSSION TITLE
Refreshing a snake on its page removes area standings

### DISCUSSION THREAD

**Discussion by md-hexdrive** [BattlesnakeOfficial/feedback#106] [Bug Reports]
When I refresh a snake from its webpage, the snake's area standings disappear on that page.

**Comment by bvanvugt** [comment]
I've been able to replicate this personally, feels like caching issue. I'll file the bug, however this is likely a low priority fix. Thanks for reporting @md-hexdrive :)

**Reply by md-hexdrive** [reply]
Glad to help! 👍

**Comment by chris-bsnake** [comment]
This fix for this bug is now live!
